### PR TITLE
[ES] Update `Client/Interface`

### PR DESCRIPTION
**Pull request by UnknownOsu**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Client/Interface/es.md [hunk 1]
```diff
@@ -297,35 +255,35 @@ A continuación se muestran las pantallas de resultados de los otros modos de ju
...
-| [`Puntuación total`](/wiki/Gameplay/Score/Total_score) | No se tiene en cuenta, ya que no afecta a su posición en las clasificaciones en línea. | Igual que la puntuación rankeada, pero tiene en cuenta todos los mapas disponibles en el sitio web de osu!, incluidos los mapas infrautilizados o fallidos. Esto cuenta para su [nivel](/wiki/Gameplay/Score/Total_score#level). |
-| [`Performance`](/wiki/Performance_points) | La cantidad de [pp no ponderados](/wiki/Performance_points#why-didn't-i-gain-the-full-amount-of-pp-from-a-map-i-played?) que recibirá por la jugada. | Su cantidad total de puntos de rendimiento, y cuánto pp valió la jugada enviada. |
+| `General` | Tu posición en la tabla de clasificación del mapa. Ten en cuenta que las puntuaciones con [mods](/wiki/Gameplay/Game_modifier) también aparecen en esta misma tabla de clasificación. | Tu [clasificación global](/wiki/Ranking#performance-points-ranking) contra todos en el mundo. |
+| [`Precisión`](/wiki/Gameplay/Accuracy) | Con qué precisión jugaste el beatmap. Esto solo se cuenta cuando se supera tu puntuación anterior. |
```
Comment (agatemosu) [inline wiki/Client/Interface/es.md:277]:
```suggestion
| [`Precisión`](/wiki/Gameplay/Accuracy) | Con qué precisión jugaste el beatmap. Esto solo se cuenta cuando superas tu puntuación anterior. | Un promedio ponderado de la precisión de tus mejores puntuaciones. |
```

File: wiki/Client/Interface/es.md [hunk 2]
```diff
@@ -297,35 +255,35 @@ A continuación se muestran las pantallas de resultados de los otros modos de ju
...
-| [`Puntuación total`](/wiki/Gameplay/Score/Total_score) | No se tiene en cuenta, ya que no afecta a su posición en las clasificaciones en línea. | Igual que la puntuación rankeada, pero tiene en cuenta todos los mapas disponibles en el sitio web de osu!, incluidos los mapas infrautilizados o fallidos. Esto cuenta para su [nivel](/wiki/Gameplay/Score/Total_score#level). |
-| [`Performance`](/wiki/Performance_points) | La cantidad de [pp no ponderados](/wiki/Performance_points#why-didn't-i-gain-the-full-amount-of-pp-from-a-map-i-played?) que recibirá por la jugada. | Su cantidad total de puntos de rendimiento, y cuánto pp valió la jugada enviada. |
+| `General` | Tu posición en la tabla de clasificación del mapa. Ten en cuenta que las puntuaciones con [mods](/wiki/Gameplay/Game_modifier) también aparecen en esta misma tabla de clasificación. | Tu [clasificación global](/wiki/Ranking#performance-points-ranking) contra todos en el mundo. |
```
Comment (agatemosu) [inline wiki/Client/Interface/es.md:276]:
```suggestion
| `General` | Tu posición en la tabla de clasificación del mapa. Ten en cuenta que las puntuaciones con [mods](/wiki/Gameplay/Game_modifier) también aparecen en esta misma tabla de clasificación. | Tu [clasificación global](/wiki/Ranking#clasificación-por-puntos-de-rendimiento) contra todos en el mundo. |
```

File: wiki/Client/Interface/es.md [hunk 3]
```diff
@@ -39,38 +33,38 @@ La rocola tiene los siguientes botones:
...
 ## Sala multijugador
 
 *Página principal: [Multijugador](/wiki/Client/Interface/Multiplayer)*
 
 ![](img/multi-lobby-ES.jpg "Sala multijugador")
 
-La sala multijugador le permite jugar con y contra otros jugadores.
+La sala multijugador te permite jugar con y contra otros jugadores.
 
 ## Selección de canciones
```
Comment (agatemosu) [inline wiki/Client/Interface/es.md:69]: Selector de canciones

File: wiki/Client/Interface/es.md [hunk 4]
```diff
@@ -85,128 +79,92 @@ El icono del [modo de juego](/wiki/Game_mode) actualmente seleccionado se puede
...
+| `Por Duración` | Los beatmaps se ordenarán por duración, del más corto al más largo. |
+| `Por Rango obtenido` | Los beatmaps se ordenarán por el rango más alto alcanzado en ellos, de peor a mejor. |
+| `Por Título` | Los beatmaps se ordenarán alfabéticamente por el título de la canción. |
 
 ### Buscar
 
-![](img/search-bar-ES.jpg)
-
-*Nota: No puede tener la consola del chat o la barra lateral de opciones abiertas si desea buscar. De lo contrario, todo lo que escriba se percibirá como texto de chat o como una consulta de búsqueda de opciones.*
+Artículo principal: [Búsqueda de beatmaps](/wiki/Beatmap_search)
```
Comment (cl8n) [inline wiki/Client/Interface/es.md:147]:
```suggestion
*Artículo principal: [Búsqueda de beatmaps](/wiki/Beatmap_search)*
```

**Review by cl8n — CHANGES_REQUESTED** [review]

**Review by cl8n — APPROVED** [review]

**Pull request merged**